src/main/java/cz/muni/fi/pv242/rest/model/Book.java

```diff
@@ -1,7 +0,0 @@
-package cz.muni.fi.pv242.rest.model;
-
-/**
- * Created by honza on 5/18/16.
- */
-public class Book {
-}
```

Comment:
Kontroloval jsi ze ty se ty entity nikde nepouzivaji a nebudou chybet?
@Dudys koukni na to prosim taky nepouziva se to nikde?